phoenix-pherf/config/scenario/timeout_test_scenario.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Same here. Move to IT directory since your IT uses this. You'll have to create the directory structure like the resources dir that's present in the test directory.